core-bundle/src/Repository/RememberMeRepository.php

```diff
@@ -18,8 +18,6 @@
 use Symfony\Component\Security\Core\Exception\TokenNotFoundException;
 
 /**
- * @template-extends ServiceEntityRepository<RememberMe>
```

Comment:
This does not seam related to this PR?

Reply:
The CI will fail without removing this:

```
PHPDoc tag @extends contains generic type Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository<Contao\CoreBundle\Entity\RememberMe> but class  
         Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository is not generic.              
```

Reply:
Interesting, removing this line breaks phpstan on my system with the following error: `Class Contao\CoreBundle\Repository\RememberMeRepository extends generic class Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository but does not specify its types: T`

Reply:
I think I know why. This was fixed in the doctrine bundle yesterday, see https://github.com/doctrine/DoctrineBundle/commit/251cd5aaea32bb92cdad4204840786b317dcdd4c

So this has to be reverted, otherwise the CI will fail on the next run (with doctrine-bundle version 2.8.2).

Reply:
Reverted in a3377eb096d354b0e242c51ee00b0816653ccffa.